clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/servergroup/DisableDcosServerGroupAtomicOperation.groovy

```diff
@@ -34,7 +35,7 @@ class DisableDcosServerGroupAtomicOperation implements AtomicOperation<Void> {
     def resizeDesc = new ResizeDcosServerGroupDescription().with {
       region = description.region
       serverGroupName = description.serverGroupName
-      instances = 0
+      capacity = new ServerGroup.Capacity(desired: 0)
```

Comment:
I don't really like the idea of massaging the capacity into a field for some reason. Sure we don't care about min/max, but in some cases neither do other providers (without advanced capacity setting, providers set min/max/desired to all be equal).